docs/bill_of_materials.md

```diff
@@ -50,19 +50,28 @@ You need a WiFi solution for each JetBot.  Select an option and get the parts sp
 
 #### Option 1 (default) - M2 Card + Antennas
 
+This is the good option for Jetson Nano (4GB) Developr Kit.<br>
+If you are using Jetson Nano 2GB Developer Kit, you need to use the Option 2 below, as the carrier board of the 2GB Dev Kit does not have the M.2 slot.
+
 |  **Part** | **Quantity** | **Cost** | **URL** | **Notes** |
 | --- | --: | --: | --- | --- |
 | WiFi card | 1 | $18.95 | [Amazon](https://amzn.to/2WKEkum), [NewEgg](https://www.neweggbusiness.com/Product/Product.aspx?Item=9SIV21M85N2699) | M2, Intel Wireless-AC 8265 |
 | WiFi antenna | 2 | $5.06 | [Arrow](https://www.arrow.com/en/products/2042811100/molex) | U.FL connectors |
 
 #### Option 2 - USB Dongle
 
-???+ bug 
-    Notice (5/17/2019):  We're currently tracking an issue with the Linux driver for the below USB WiFi dongle that may cause connectivity issues.  You can find information in [this forum post](https://devtalk.nvidia.com/default/topic/1049303/jetson-nano/jetson-nano-wifi-/post/5329699/#5329699).  For now, we recommend using Option 1 listed above.  Thank you to those who discovered this issue!
+If you are using Jetson Nano 2GB Developer Kit, you can either use the USB Wi-fi dongle that is bundled (depending on the region), or one of the USB wifi dongle listed below.<br>
```

Comment:
This option is compatible with both the original Jetson Nano (4GB) and Jetson Nano 2GB.